nixos/modules/services/monitoring/prometheus/exporters/smartctl.nix

```diff
@@ -41,13 +41,23 @@ in {
   serviceOpts = {
     serviceConfig = {
       AmbientCapabilities = [
+        "CAP_RAW_SYSIO"
         "CAP_SYS_ADMIN"
       ];
       CapabilityBoundingSet = [
+        "CAP_RAW_SYSIO"
         "CAP_SYS_ADMIN"
       ];
       DevicePolicy = "closed";
-      DeviceAllow = lib.mkForce cfg.devices;
+      DeviceAllow = lib.mkOverride 100 (
```

Comment:
This should all overrides with `lib.mkForce` which uses a priority of 50.

Reply:
The description of the devices option should maybe be adjusted to reflect that not setting it will do some kind of autodiscovery.

Reply:
Oh yeah, totally. We even talked about that.